pages/docs/data-modeling/description-syncing.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout } from "nextra-theme-docs";
+
+# Description Syncing
+
+If you've added descriptions to your columns via column comments/descriptions or
+DBT `persist-docs`, Glean will automatically import those descriptions when you
+create or update a Data Model.
```

Comment:
is it true that whenever i update a model, the descriptions get synced? i think its always an explicit user action?

Reply:
good catch!